<ui version="4.0" >
 <class>SearchBanner</class>
 <widget class="QWidget" name="SearchBanner" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>36</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="spacing" >
    <number>-1</number>
   </property>
   <property name="leftMargin" >
    <number>12</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>26</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="searchInfo" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="previousButton" >
     <property name="text" >
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextButton" >
     <property name="text" >
      <string>></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SearchLineEdit" name="searchLineEdit" />
   </item>
   <item>
    <widget class="QPushButton" name="doneButton" >
     <property name="text" >
      <string>Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>QLineEdit</extends>
   <header>searchlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
